tests/monitoring_test.go

```diff
@@ -0,0 +1,128 @@
+package tests
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+var _ = Describe("Monitoring Tests", func() {
+	fw := framework.NewFramework("monitoring-test")
+
+	BeforeEach(func() {
+		if !fw.IsPrometheusAvailable() {
+			Skip("This test depends on prometheus infra being available")
+		}
+	})
+
+	Context("Alerts", func() {
+		It("should throw CDIDataVolumeUnusualRestartCount alert when a DataVolume object restarts more than three times", func() {
+			By("creating a DataVolume")
+			dataVolume := utils.NewDataVolumeWithHTTPImport(dataVolumeName, "500Mi", validURL)
+			dataVolume, err := utils.CreateDataVolumeFromDefinition(fw.CdiClient, fw.Namespace.Name, dataVolume)
+			Expect(err).ToNot(HaveOccurred())
+			fw.ForceBindPvcIfDvIsWaitForFirstConsumer(dataVolume)
+
+			By("waiting for the DataVolume to be restarted more than three times")
+			Eventually(func() bool {
```

Comment:
Eventually without `.Should(BeSomething())`? What am I missing

Reply:
it was missing